.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
+-Pmight-produce-incrementals
```

Comment:
Enables [incremental builds](https://github.com/jenkinsci/incrementals-tools) in CI.